lib/client.js

```diff
@@ -75,7 +77,11 @@ module.exports = function (classes){
 
       if (_.has(command, 'error') && (!_.isNull(command['error']) && !_.isEmpty(command['error']))) {
         // we have an error, we need to act on that, regardless of other members in the command received
-        throw new Error(command);
+        throw Object.assign(new Error(command.error[1]), {

```

Comment:
btw, tiny tip, you can update version using this:
https://docs.npmjs.com/cli/version
it will create a nice version tag and will commit it.
makes package updating a little more consistent :)